src/main/cluster.ts

```diff
@@ -110,7 +111,7 @@ export class Cluster implements ClusterModel {
 
   protected bindEvents() {
     logger.info(`[CLUSTER]: bind events`, this.getMeta());
-    const refreshTimer = setInterval(() => this.online && this.refresh(), 30000); // every 30s
+    const refreshTimer = setInterval(() => !this.disconnected && this.refresh(), 30000); // every 30s
```

Comment:
We want to refresh connection status even when cluster is offline/not accessible.